basket/news/tests/test_tasks.py

```diff
@@ -1019,3 +1020,49 @@ def test_basetime_provided(self):
         basetime = datetime.fromtimestamp(1473428035.498)
         formatted_time = gmttime(basetime)
         self.assertEqual(formatted_time, 'Fri, 09 Sep 2016 13:43:55 GMT')
+
+
+@patch('basket.news.tasks.sfdc')
+@patch('basket.news.tasks.get_user_data')
+class CommonVoiceGoalsTests(TestCase):
+    def test_new_user(self, gud_mock, sfdc_mock):
+        gud_mock.return_value = None
+        data = {
+            'email': 'dude@example.com',
+            'first_contribution_date': '2018-06-27T14:56:58Z',
+            'last_active_date': '2019-07-11T10:28:32Z',
+            'two_day_streak': False,
+        }
+        record_common_voice_goals(data)
+        # ensure passed in dict was not modified in place.
+        # if it is modified a retry will use the modified dict.
+        assert data == data
```

Comment:
oh wow. you and I both missed this. now I feel dumb. this isn't testing anything 🤦‍♂ 

I fixed it in 4d0668d4e5cb304e68b04bcc0d2b278d27b591e6

Reply:
lol. my mind knew the intent! 🤦‍♂